python/lib/dependabot/python/file_updater/poetry_file_updater.rb

```diff
@@ -204,7 +203,7 @@ def run_poetry_update_command
         end
 
         def run_poetry_command(command, fingerprint: nil)
-          Helpers.run_poetry_command(command, fingerprint: fingerprint)
+          SharedHelpers.run_shell_command(command, fingerprint: fingerprint)
```

Comment:
Same as the other comment, but reversed. This used to raise an error including only stderr, now we'll raise an error include stdout and stderr concatenated.

Seems fine to I guess!